Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -19006,7 +19014,6 @@
 			isa = PBXTargetDependency;
 			platformFilters = (
 				ios,
-				xros,
```

Comment:
I don't know why these were removed. Do I need to put them back?

Reply:
Yes. Xcode fights about this when you don't have the Vision Pro SDK installed, but they need to be put back in.

Reply:
Yes. Don't delete those